VideoLocker/res/layout/row_myvideo_course_list.xml

```diff
@@ -53,62 +59,40 @@
                 android:textColor="@color/org_code_txt"
                 android:textSize="12sp" />
 
-            <LinearLayout
+            <org.edx.mobile.view.custom.ETextView
+                android:id="@+id/no_of_videos"
+                style="@style/regular_grey_text"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_alignParentBottom="true"
-                android:layout_alignParentRight="true"
-                android:layout_alignParentEnd="true"
-                android:layout_centerHorizontal="true"
-                android:layout_marginLeft="5dp"
```

Comment:
Currently it has no left margin - you removed it. Even if there is space, there should still be some margin between views.